staging/src/k8s.io/apiserver/pkg/server/options/serving_test.go

```diff
@@ -43,7 +43,6 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/version"
 	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
-	"k8s.io/apiserver/pkg/server"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#import-dot do we actually need the dot import? If not I think we are better off eliminating line 47 and keeping 46.

Reply:
It's my fault. I reverted this one.